lib/tests/player-test.js

```diff
@@ -0,0 +1,36 @@
+import Player from '../src/app/player';
```

Comment:
wouldn't it be better to keep tests in one directory with the module?

Reply:
I don't know. I made separate tests directory since we have it that way in skin, core and html5 player

Reply:
as you set up tests from scratch we can discuss it 
@ooyala/player-web